pt_PT/LC_MESSAGES/unreadComments.po

```diff
@@ -1,22 +1,22 @@
 msgid ""
 msgstr ""
-"Language: en_US\n"
+"Language: pt_PT\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 
 msgid "LOGIN_REQUIRED_TO_VIEW_PAGE"
-msgstr "You must log in to view this page."
+msgstr "Tens de ter feito login para aceder a esta página."
 
 msgid "HEADER_UNREAD_COMMENTS"
-msgstr "Unread Comments"
+msgstr "Comentários Não Lidos"
 
 msgid "NO_UNREAD_COMMENTS"
-msgstr "You have no unread comments."
+msgstr "Não tens comentários por ler."
 
 msgid "TAP_ON_FLIPNOTE_ID_TO_VISIT_COMMENTS"
-msgstr "Tap on a Flipnote ID to visit the comments page."
+msgstr "Toque no ID do Flipnote para visitar a página de comentários."
 
 msgid "FLIPNOTE_ID"
-msgstr "Flipnote ID"
+msgstr "ID do Flipnote"
 
 msgid "UNREAD"
-msgstr "Unread"
+msgstr "N"
```

Comment:
Is this correct?

Reply:
damn it, unfortunately no, i must've skipped it mid translation, hopefully i can fix it

Reply:
No problem, you can commit the fix to this branch in your repository and it will be a part of this pull request.

Reply:
well this sucks, it's not letting me fix it

Reply:
hopefully i managed to fix it

Reply:
All fixed!